hash/megadriv.xml

```diff
@@ -16896,12 +16942,12 @@ https://segaretro.org/Feng_Shen_Ying_Jie_Chuan
 	</software>
 
 	<software name="gamenko">
-		<description>Game no Kandume Otokuyou (Japan)</description>
+		<description>Game no Kanzume Otokuyou (Japan, Sega Channel)</description>
```

Comment:
I feel like `Sega Channel` is redundant here.

Reply:
Should I remove all instances of "Sega Channel" in the descriptions and add a comment instead?